apps/api/src/app/topics-v2/dtos/create-topic-subscriptions.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ArrayMaxSize, IsArray, IsDefined } from 'class-validator';
+
+export class CreateTopicSubscriptionsRequestDto {
+  @ApiProperty({
+    description: 'List of subscriber identifiers to subscribe to the topic (max: 100)',
+    example: ['subscriberId1', 'subscriberId2'],
+    type: [String],
+  })
+  @IsArray()
+  @IsDefined()
+  @ArrayMaxSize(100, { message: 'Cannot subscribe more than 100 subscribers at once' })
```

Comment:
do we want to add ArrayMaxSize as well? i an not sure if those validation will validate `[]`

Reply:
You min min size? good call